workos/user_management.py

```diff
@@ -207,49 +208,125 @@ def update_user_password(self, user_id, password):
+        headers = {}
+
+        response = self.request_helper.request(
+            ORGANIZATION_MEMBERSHIP_DETAIL_PATH.format(organization_membership_id),
+            method=REQUEST_METHOD_GET,
+            headers=headers,
+            token=workos.api_key,
+        )
 
-    def remove_user_from_organization(self, user_id, organization_id):
-        """Removes a User from the given Organization.
+        return WorkOSOrganizationMembership.construct_from_response(response).to_dict()
+
+    def list_organization_memberships(
+        self,
+        user_id=None,
+        organization_id=None,
+        limit=None,
+        before=None,
+        after=None,
+        order=None,
+    ):
+        """Get a list of all of your existing organization memberships matching the criteria specified.
 
         Kwargs:
-            user_id (str): The unique ID of the User.
-            organization_id (str): Unique ID of the Organization.
+            user_id (str): Filter Organization Memberships by user. (Optional)
+            organization_id (str): Filter Organization Memberships by organization. (Optional)
+            limit (int): Maximum number of records to return. (Optional)
+            before (str): Pagination cursor to receive records before a provided Organization Membership ID. (Optional)
+            after (str): Pagination cursor to receive records after a provided Organization Membership ID. (Optional)
+            order (Order): Sort records in either ascending or descending order by created_at timestamp: "asc" or "desc" (Optional)
 
         Returns:
-            dict: User response from WorkOS.
+            dict: Organization Memberships response from WorkOS.
         """
 
-        headers = {}
+        default_limit = None
+
+        if limit is None:
+            limit = RESPONSE_LIMIT
+            default_limit = True
+
+        params = {
+            "user_id": user_id,
+            "organization_id": organization_id,
+            "limit": limit,
+            "before": before,
+            "after": after,
+        }
+
+        if order is not None:
+            if isinstance(order, Order):
+                params["order"] = str(order.value)
+            elif order == "asc" or order == "desc":
+                params["order"] = order
+            else:
+                raise ValueError("Parameter order must be of enum type Order")
 
         response = self.request_helper.request(
-            USER_ORGANIZATION_PATH.format(user_id, organization_id),
-            method=REQUEST_METHOD_DELETE,
-            headers=headers,
+            ORGANIZATION_MEMBERSHIP_PATH,
+            method=REQUEST_METHOD_GET,
+            params=params,
             token=workos.api_key,
         )
 
-        return WorkOSUser.construct_from_response(response).to_dict()
+        response["metadata"] = {
+            "params": params,
+            "method": UserManagement.list_organization_memberships,
+        }
+
+        if "default_limit" in locals():
```

Comment:
lol woof, i know this is copy/pasted and not user-facing so we shouldn't get distracted by this, but this technique could use some work 😆  `default_limit` will always be present in `locals()` since it is set up here 🤦‍♀️  https://github.com/workos/workos-python/pull/218/files#diff-87ef5545759351bbeeba22da6f30add44d7c735f751c4807fd059d6ef87931c6R277

Reply:
let's ignore for now! 🙃 🙂 🙃 🙂 🙃 🙂 🙃 🙂

Reply:
I'll make a note to look at it in more detail once we get the UM API out 👍

Reply:
🙌 happy to look at it with you when the time comes!